scopesim_templates/metis/laser.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""WCU laser spectrum for METIS."""
+
+import numpy as np
+from astropy import units as u
+from astropy.io import fits
+
+from spextra import Spextrum
+
+from ..rc import Source, im_plane_utils
+
+
+def laser_spectrum(img_size=(12, 12),
+                   waves=np.arange(3, 6, .001)*u.um,
+                   centers=[3.39, 4.73, 5.263]*u.um,
+                   fwhms=[8E-8, 8E-7, 6E-5]*u.um,
+                   fluxes=[20, 1, 22]):
+    """
+    Create laser calibration spectrum source for METIS.
+
+    Parameters
+    ----------
+    img_size : TYPE, optional
+        FOV size in arcsec, plus a little extra. The default is (12, 12).
+    waves : TYPE, optional
+        Input wavelength axis. The default is np.arange(3, 6, .001)*u.um.
+    centers : TYPE, optional
+        Emission line centers. The default is [3.39, 4.73, 5.263]*u.um.
+    fwhms : TYPE, optional
+        Emission line FWHMs. The default is [8E-8, 8E-7, 6E-5]*u.um.
+    fluxes : TYPE, optional
+        Emission line total fluxes. The default is [20, 1, 22] to create lines
+        of roughly equal amplitude.
```

Comment:
I don't understand why `[20, 1, 22]` would give equal fluxes. I would expect the middle value to be in between the other two values. Also what units are these?